src/tls/Makefile.common

```diff
@@ -16,7 +16,10 @@ INCLUDE_PATHS 	= \
   $(QD_HOME)/src/lowparse \
   $(MITLS_HOME)/src/parsers/generated \
   $(MITLS_HOME)/src/tls \
-  $(MITLS_HOME)/src/tls/obj
```

Comment:
do you remember or understand why obj needs to be on the include path?

Reply:
I think this is only needed to benefit from the cache in interactive mode. I think we can add it only in `-in` targets, but I'm not absolutely sure.

Reply:
We can. I removed it.